account_credit_control_dunning_fees/tests/test_fees_generation.py

```diff
@@ -16,7 +16,16 @@ def setUp(self):
         self.assertTrue(self.usd)
 
         self.company = self.browse_ref('base.main_company')
-        self.company.currency_id = self.euro
+        # travis fails
+        if self.company.currency_id != self.euro:
+            self.env.cr.execute("""
+                UPDATE res_company
+                SET currency_id = %(currency)s
+                WHERE id = %(company)s
+                """, {'currency': self.euro.id,
+                      'company': self.company.id})
+            self.env.invalidate(self.env['res.company']._fields['currency_id'],
+                                self.company.ids)
```

Comment:
Sorry, I messed up. The argument to invalidate is actually a tuple (field, ids).

Reply:
A *list* of such tuples...